tools-reference/bash/text.xml

```diff
@@ -119,6 +119,14 @@ The <c>[[ ]]</c> form is generally safer than <c>[ ]</c> and should be used in
 all new code.
 </important>
 
+<p>
+Note that POSIX-compliance is not a concern for ebuilds themselves as bash
+is guaranteed to be present and ebuilds are written in bash. Therefore, authors
+should use the canonical standard tests to adhere to the principle of
+least surprise. This aids review and ensures that use is consistent throughout
+the repository.
+</p>
+
```

Comment:
Sorry, but IMHO this paragraph has horrible style. Suggestion for first sentence: "POSIX compliance is not a concern for ebuilds, as their interpreter is guaranteed to be GNU Bash."

It's not entirely clear to me what you want to say with the second sentence?

Reply:
For reference, it sounds a bit better to say "How about ...?" or "This may be clearer: ..." rather than just "horrible", i.e. focus on the actual improvement. But I know it wasn't intended badly.

How about now?

Reply:
Sorry if you perceived my wording as rude, that wasn't my intention. Cultural difference, I guess?

Reply:
No worries at all, I know it wasn't! 🙂